src/System.Collections/src/System/Collections/Generic/Stack.cs

```diff
@@ -258,15 +258,15 @@ public T[] ToArray()
         public struct Enumerator : IEnumerator<T>,
             System.Collections.IEnumerator
         {
-            private Stack<T> _stack;
+            private readonly Stack<T> _stack;
             private int _index;
-            private int _version;
+            private readonly int _version;
```

Comment:
Nit: Consider swapping the _index and _version lines, so that the readonly state is grouped together.